mne/epochs.py

```diff
@@ -1643,7 +1643,7 @@ def tmax(self):
         """Last time point."""
         return self.times[-1]
 
-    def __repr__(self):
+    def __repr__(self, limit_events=10):
```

Comment:
you cannot have a param in a __repr__ AFAIK

Reply:
Keywords seem to work, but I dont know if thats intended or not.
Alternatives that come to mind:
1) simply set `max_events` inside the function, disabling customization of the output
or
2) Somewhere in BaseEpochs add an attribute `repr_verboseness` or something and infer `max_events` from there on.